include/ultra64/libc.h

```diff
@@ -5,7 +5,7 @@
 
 void osSyncPrintf(const char* fmt, ...);
 
-void bzero(void* __s, size_t __n);
+void bzero(void* __s, int __n);
 int bcmp(const void* __sl, const void* __s2, size_t __n);
 void bcopy(const void* __src, void* __dest, size_t __n);
```

Comment:
Shouldn't all these be `int` instead of only `bzero`?

Reply:
Yes, since it didn't directly apply to what I was doing I didn't touch them. However I'll just fix them now.